pkg/handlers/activate_test.go

```diff
@@ -125,10 +127,12 @@ func (suite *ActivateTestSuite) TestTrackFeatureWithFeatureRollout() {
 	suite.NoError(err)
 
 	expected := optimizely.Decision{
-		UserID:     "testUser",
-		FeatureKey: "one",
-		Enabled:    true,
-		Type:       "feature",
+		UserID:        "testUser",
+		FeatureKey:    "one",
+		Enabled:       true,
+		Type:          "feature",
+		ExperimentKey: "1",
+		VariationKey:  "3",
```

Comment:
As mentioned in the Go SDK PR: I don't think it should return `ExperimentKey` and `VariationKey` when the user was bucketed into a rollout (not a feature test). The keys of rollout experiments are not exposed or meaningful to the user.